app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/onboarding/email/email_verified.scala.html

```diff
@@ -27,7 +27,7 @@
   mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
   pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
   accountMenu: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.account_menu,
-  caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption,
+  caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption_govuk,
```

Comment:
I think, we need to use different approach. you can fix it using PageHeading component like 
`label = pageHeading(title, Some(messages("returns.disposal-details.caption"))),`

try this one if works okay. otherwise there is another approach too.